### DISCUSSION TITLE
Adding more options like SD Maid 1

### DISCUSSION THREAD

**Discussion by rapdodge** [d4rken-org/sdmaid-se#935] [Ideas]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

> Not really, but it's nice to having it.

**Describe the solution you'd like**

> Adding more options like SD Maid 1 (will be attached)

**Describe alternatives you've considered**

> I don't know, maybe already the bestest option.

**Describe why this would be in the interest of all users**
Describe why this is not a niche feature, but would be a benefit for the majority of users.

> I don't know, but, I'm using it, it's nice feature when you wanted to having more control about applications

**Additional context**

<details>
	<summary>
		Things that nice to add into SD Maid SE
	</summary>
The first things, is the three dot things on the SD Maid 1, that has "Kill application", "Force stop application", "Export", "Reset", and "Share". Eventhough, mostly, I'm just using kill, force stop, and reset application.

![image](https://github.com/d4rken-org/sdmaid-se/assets/33513626/42ee3480-0a51-46fd-864e-dc14fef5b877)

Meanwhile, SD Maid SE didn't have that.

![image](https://github.com/d4rken-org/sdmaid-se/assets/33513626/56016c73-a080-42e9-a96e-1992dd17a74f)

And on SD Maid 1, there is more features like "Permission lists", "Activity manager", "Force stop application", "Receiver Manager", and "Reset"

![image](https://github.com/d4rken-org/sdmaid-se/assets/33513626/7d90d974-a0f6-4937-b909-a811ecada370)

Oh, yes, and the application lists, there is app state too... It's nice indicator too, so, the app that doesn't want to running can be stop too by batch.
</details>

**Comment by Jeffhaji6** [comment]
> **Is your feature request related to a problem? Please describe.**
> A clear and concise description of what the problem is.
> 
> > Not really, but it's nice to having it.
> 
> **Describe the solution you'd like**
> A clear and concise description of what you want to happen.
> 
> > Adding more options like SD Maid 1 (will be attached)
> 
> **Describe alternatives you've considered**
> A clear and concise description of any alternative solutions or features you've considered.
> 
> > I don't know, maybe already the bestest option.
> 
> **Describe why this would be in the interest of all users**
> Describe why this is not a niche feature, but would be a benefit for the majority of users.
> 
> > I don't know, but, I'm using it, it's nice feature when you wanted to having more control about applications
> 
> **Additional context**
> Add any other context or screenshots about the feature request here.
> 
> <details>
> 	<summary>
> 		Things that nice to add into SD Maid SE
> 	</summary>
> The first things, is the three dot things on the SD Maid 1, that has "Kill application", "Force stop application", "Export", "Reset", and "Share". Eventhough, mostly, I'm just using kill, force stop, and reset application.
> 
> ![image](https://github.com/d4rken-org/sdmaid-se/assets/33513626/42ee3480-0a51-46fd-864e-dc14fef5b877)
> 
> Meanwhile, SD Maid SE didn't have that.
> 
> ![image](https://github.com/d4rken-org/sdmaid-se/assets/33513626/56016c73-a080-42e9-a96e-1992dd17a74f)
> 
> And on SD Maid 1, there is more features like "Permission lists", "Activity manager", "Force stop application", "Receiver Manager", and "Reset"
> 
> ![image](https://github.com/d4rken-org/sdmaid-se/assets/33513626/7d90d974-a0f6-4937-b909-a811ecada370)
> 
> Oh, yes, and the application lists, there is app state too... It's nice indicator too, so, the app that doesn't want to running can be stop too by batch.
> </details>
